plugin/adsk/scripts/mayaUsd_exportHelpers.py

```diff
@@ -33,7 +41,7 @@ def updateDefaultPrimCandidates(excludeMesh, excludeLight, excludeCamera):
 
         allItems = list(set(allItems) - set(startup_cameras))
     
-    allItems = list(set(allItems) - set(excludeList))
+    allItems = removeHiddenInOutliner(list(set(allItems) - set(excludeList)))
```

Comment:
Feels like there's an easier way to just check if hidden in L19 when we query the item list. In python if you do set(a) - set(b), and b has items a doesn't have it will just ignore them

Reply:
The reason I do it here is that there will be fewer items to check after we removed the ones we already don't want.

Reply:
Good point!